package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "coam-client",
-  "version": "0.2.3",
```

Comment:
Awesome, could you please update the CHANGELOG as well ?

Reply:
Sure, updated it